plugins/module_utils/network/nxos/config/acls/acls.py

```diff
@@ -656,7 +656,7 @@ def set_commands(self, want, have):
                                         # remove existing rule to update it
                             elif w_acl.get("aces"):
                                 # 'have' has ACL defined without any ACE
-                                ace_list = [item for item in w_acl["aces"]]
+                                ace_list = list(w_acl["aces"])
```

Comment:
Are this changes related to fixing sanity?

Reply:
Yeah, pylint was failing with `unnecessary use of comprehension`.